assets/src/components/stacks/variables/StackVariables.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Input,
+  SearchIcon,
+  Table,
+  useSetBreadcrumbs,
+} from '@pluralsh/design-system'
+import { useMemo, useState } from 'react'
+import { useOutletContext } from 'react-router-dom'
+import { useTheme } from 'styled-components'
+import { createColumnHelper } from '@tanstack/react-table'
+import { useDebounce } from '@react-hooks-library/core'
+import { FullHeightTableWrap } from 'components/utils/layout/FullHeightTableWrap'
+
+import { StackOutletContextT, getBreadcrumbs } from '../Stacks'
+
+const columnHelper = createColumnHelper<{
+  name: string
+  value: Nullable<string>
+}>()
+
+const columns = [
+  columnHelper.accessor((row) => row.name, {
+    id: 'name',
+    header: 'Name',
+    enableGlobalFilter: true,
+    meta: { truncate: true },
+    cell: ({ getValue }) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      const theme = useTheme()
+
+      return <span css={{ ...theme.partials.text.code }}>{getValue()}</span>
+    },
+  }),
+  columnHelper.accessor((row) => row.value, {
+    id: 'value',
+    header: 'Value',
+    meta: { truncate: true },
+    cell: ({ getValue }) => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      const theme = useTheme()
+
+      return <span css={{ ...theme.partials.text.code }}>{getValue()}</span>
```

Comment:
This can technically be any type, including an object, wondering if we should json stringify it?

Reply:
Right, I've done it.